### PR TITLE
adds the giga drill to mining vendor for 50000 mining points

### PR DESCRIPTION
**Pull request by boodaliboo**
gigadrill is incredibly overpowered but has no way currently to be spawned but by admins which wont likely ever happen as it isnt intresting outside of how good it mines. 50 thousand points is a insane ammount of mining points and if someone gets that much they deserve the drill
#### Changelog

:cl:  
rscadd: Added giga drill to mining vendor 
/:cl:

### CODE REVIEW

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -66,8 +66,9 @@
 		new /datum/data/mining_equipment("Cigar",						/obj/item/clothing/mask/cigarette/cigar/havana,						150, VENDING_MISC),
 		new /datum/data/mining_equipment("Soap",						/obj/item/soap/nanotrasen,											200, VENDING_MISC),
 		new /datum/data/mining_equipment("Laser Pointer",				/obj/item/laser_pointer,											300, VENDING_MISC),
-		new /datum/data/mining_equipment("Space Cash",					/obj/item/stack/spacecash/c1000,									2000, VENDING_MISC)
-	)
+		new /datum/data/mining_equipment("Space Cash",					/obj/item/stack/spacecash/c1000,									2000, VENDING_MISC),
+		new /datum/data/mining_equipment("Giga Drill",					/obj/vehicle/ridden/gigadrill,									1000000, VENDING_MISC)
+
```
Comment (ghost) [inline code/modules/mining/machine_vending.dm:72]: One more closing parentheses and you're good to go

**Comment by ghost** [comment]
50k mining points. At 1 million there will be nothing left to mine, at 50k points, there's enough material that mining any more extra won't make too much of a difference. If not 50, then whatever is a decently achievable but difficult amount of points.

**Comment by Mesalikepie** [comment]
New objective once I leave LOA. Get this

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -66,8 +66,10 @@
 		new /datum/data/mining_equipment("Cigar",						/obj/item/clothing/mask/cigarette/cigar/havana,						150, VENDING_MISC),
 		new /datum/data/mining_equipment("Soap",						/obj/item/soap/nanotrasen,											200, VENDING_MISC),
 		new /datum/data/mining_equipment("Laser Pointer",				/obj/item/laser_pointer,											300, VENDING_MISC),
-		new /datum/data/mining_equipment("Space Cash",					/obj/item/stack/spacecash/c1000,									2000, VENDING_MISC)
-	)
+		new /datum/data/mining_equipment("Space Cash",					/obj/item/stack/spacecash/c1000,									2000, VENDING_MISC),
+		new /datum/data/mining_equipment("Giga Drill",					/obj/vehicle/ridden/gigadrill,									50000, VENDING_MISC)
+		)
```
Comment (jcatjmeow) [inline code/modules/mining/machine_vending.dm]:
```suggestion
	)
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

**Comment by alexkar598** [comment]
adding this will mark the end of getting xenoarch, sad pog but okay i guess as long as theres enough mats at that point that it doesnt matter

**Comment by boodaliboo** [comment]
> adding this will mark the end of getting xenoarch, sad pog but okay i guess as long as theres enough mats at that point that it doesnt matter

that is a lie you stopped working on xenoarch years ago

**Comment by alexkar598** [comment]
yes

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**